.ci/docker/jruby/run.sh

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env bash
 
+set -o pipefail
+
+# 7-jdk is excluded by default. See https://github.com/elastic/apm-agent-ruby/pull/1367#issuecomment-1437228929
+EXCLUDE=("7-jdk")
```

Comment:
@estolfo regarding your question.

I did not remove it, but I excluded it by default.

Reply:
Ok, so does that mean the test matrix will continue to use the last created jdk7 docker image?

Reply:
Yes, the docker image is kept in the docker registry and can still be used for tests.